Lib/test/test_ftplib.py

```diff
@@ -984,12 +984,6 @@ def test_context(self):
         ctx = ssl.SSLContext(ssl.PROTOCOL_TLS_CLIENT)
         ctx.check_hostname = False
         ctx.verify_mode = ssl.CERT_NONE
-        self.assertRaises(ValueError, ftplib.FTP_TLS, keyfile=CERTFILE,
-                          context=ctx)
-        self.assertRaises(ValueError, ftplib.FTP_TLS, certfile=CERTFILE,
-                          context=ctx)
-        self.assertRaises(ValueError, ftplib.FTP_TLS, certfile=CERTFILE,
-                          keyfile=CERTFILE, context=ctx)
```

Comment:
Why is it removed? Is it no longer error?

Reply:
For me, it's not really worth it to test that calling a function with parameters which don't exist raises an error. But ok, I updated the tests rather than removing them.